app/uk/gov/hmrc/serviceconfigs/persistence/DeploymentConfigRepository.scala

```diff
@@ -71,7 +71,12 @@ class DeploymentConfigRepository @Inject()(
         .map(_ => ())
     }
 
-  def findAll(environment: Environment): Future[Seq[DeploymentConfig]] =
+  def findAll: Future[Seq[DeploymentConfig]] =
+    collection
+      .find()
+      .toFuture()
+
+  def findAllForEnv(environment: Environment): Future[Seq[DeploymentConfig]] =
```

Comment:
or you could just make environment optional - this allows us to mix and match filters without a load of overrloadings
note: `collection.find(optEnv.fold(BsonDocument())(equal("environment", _.asString))`